src/interface_r/R/auto_generated_wrappers.R

```diff
@@ -219,199 +215,119 @@ h2o4gpu.gradient_boosting_regressor <- function(
+  model <- h2o4gpu$ElasticNetH2O(
+    n_threads = n_threads,
+    gpu_id = as.integer(gpu_id),
     n_gpus = as.integer(n_gpus),
-    glm_stop_early = glm_stop_early,
-    glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    backend = backend
-  )
-  h2o4gpu_model(model, c("classifier"))
-}
-
-#' @export
-h2o4gpu.lasso_regressor <- function(
-	alpha = 1.0,
-	fit_intercept = TRUE,
-	normalize = FALSE,
-	precompute = FALSE,
-	copy_X = TRUE,
-	max_iter = 5000L,
-	tol = 0.01,
-	warm_start = FALSE,
-	positive = FALSE,
-	random_state = NULL,
-	selection = "cyclic",
-	n_gpus = -1L,
-	glm_stop_early = TRUE,
-	glm_stop_early_error_fraction = 1.0,
-	verbose = FALSE,
-	backend = "auto") {
-
-  model <- h2o4gpu$Lasso(
-    alpha = alpha,
     fit_intercept = fit_intercept,
-    normalize = normalize,
-    precompute = precompute,
-    copy_X = copy_X,
-    max_iter = as.integer(max_iter),
+    lambda_min_ratio = lambda_min_ratio,
+    n_lambdas = as.integer(n_lambdas),
+    n_folds = as.integer(n_folds),
+    n_alphas = as.integer(n_alphas),
     tol = tol,
-    warm_start = warm_start,
-    positive = positive,
-    random_state = as_nullable_integer(random_state),
-    selection = selection,
-    n_gpus = as.integer(n_gpus),
+    tol_seek_factor = tol_seek_factor,
+    lambda_stop_early = lambda_stop_early,
     glm_stop_early = glm_stop_early,
     glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    verbose = verbose,
-    backend = backend
-  )
-  h2o4gpu_model(model, c("regressor"))
-}
-
-#' @export
-h2o4gpu.ridge_regressor <- function(
-	alpha = 1.0,
-	fit_intercept = TRUE,
-	normalize = FALSE,
-	copy_X = TRUE,
-	max_iter = 5000L,
-	tol = 0.01,
-	solver = "auto",
-	random_state = NULL,
-	n_gpus = -1L,
-	glm_stop_early = TRUE,
-	glm_stop_early_error_fraction = 1.0,
-	verbose = FALSE,
-	backend = "auto") {
-
-  model <- h2o4gpu$Ridge(
-    alpha = alpha,
-    fit_intercept = fit_intercept,
-    normalize = normalize,
-    copy_X = copy_X,
     max_iter = as.integer(max_iter),
-    tol = tol,
-    solver = solver,
-    random_state = as_nullable_integer(random_state),
-    n_gpus = as.integer(n_gpus),
-    glm_stop_early = glm_stop_early,
-    glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    verbose = verbose,
-    backend = backend
+    verbose = as.integer(verbose),
+    family = "elasticnet",
+    store_full_path = as.integer(store_full_path),
+    lambda_max = lambda_max,
+    alpha_max = alpha_max,
+    alpha_min = alpha_min,
+    alphas = alphas,
+    lambdas = lambdas,
+    double_precision = double_precision,
+    order = order
   )
   h2o4gpu_model(model, c("regressor"))
 }
 
 #' @export
-h2o4gpu.elastic_net_regressor <- function(
-	alpha = 1.0,
-	l1_ratio = 0.5,
+h2o4gpu.elastic_net_classifier <- function(
+	n_threads = NULL,
+	gpu_id = 0L,
+	n_gpus = -1L,
 	fit_intercept = TRUE,
-	normalize = FALSE,
-	precompute = FALSE,
-	max_iter = 5000L,
-	copy_X = TRUE,
+	lambda_min_ratio = 1e-07,
```

Comment:
What happened to `normalize`, does this mean that we're no longer normalizing, or normalizing by default?  I guess sklearn does this outside of ElasticNetH2O, so maybe we don't have the ability to do that within ElasticNetH2O.  That seems like a loss of good functionality... why not just utilize sklearn's ability to normalize.  Another reason that we may consider switching back t using the sklearn wrappers instead of the H2O wrappers directly... :thinking: